lib/Transforms/GenerateParam/GenerateParam.td

```diff
@@ -78,13 +78,29 @@ def GenerateParamBGV : Pass<"generate-param-bgv"> {
 def GenerateParamBFV : Pass<"generate-param-bfv"> {
   let summary = "Generate BFV Scheme Parameter";
   let description = [{
-    The pass generates the BFV scheme parameter.
+    The pass generates the BFV scheme parameter using a given noise model.
 
-    This pass get the multiplicative depth from the IR and generates the
-    moduli chain of that long consisting of 60 bits primes.
+    There are three noise models available:
+    - `bfv-noise-by-bound-coeff-average-case{-pk,-sk}`
+    - `bfv-noise-by-bound-coeff-worst-case{-pk,-sk}`
+
+    The `-pk`/`-sk` suffixes assume the input ciphertexts are
+    encrypted using the public/secret key.
+
+    The first two models are taken from KPZ21, and they work by bounding
+    the coefficient embedding of the ciphertexts. The difference
+    of the two models is expansion factor used for multiplication
+    of the coefficients, the first being `2\sqrt{N}` and the second
```

Comment:
```suggestion
    of the coefficients, the first being $2 \sqrt{N}$ and the second
```

The rendered docs have MathJAX support, so this can be rendered. Otherwise no reason to put the latex notation in the docs, could use `2 sqrt(N)`